aquarius/events/util.py

```diff
@@ -47,6 +47,24 @@ def get_network_name():
     return network_name
 
 
+def get_start_block_by_chain_id(chain_id: int) -> str:
+    """Return the contract address with the given name and chain id"""
+    with open(get_address_file(), "r") as address_json:
+        addresses = json.load(address_json)
+    return next(
```

Comment:
please add an empty line before `return`.
Also, do not forget about the doc change related to utf-8.